app/models/spotlight/page_configurations.rb

```diff
@@ -0,0 +1,91 @@
+module Spotlight
+  ##
+  # PageConfigurations is a simple class to gather and return all
+  # configurations needed for the various SirTrevor widgets.
+  # A downstream application can inject its own data like so
+  # Spotlight::PageConfigurations.local_params = {
```

Comment:
Where is the recommended place for this to happen in a Spotlight application?

Reply:
Most likely in an initializer.  Should I add that to this comment?

Reply:
Ah gotcha.. sure / maybe.. i was wondering if these were like per widget configs , but I guess they are global which is fine

Reply:
If we're putting them in an initializer, does it make sense to add them as engine configuration, just for the sake of keeping everything in one place?

Reply:
Sure, that makes sense.

Reply:
I've updated the code and relevant comment, let me know if there should be addl. changes.